linux/debian/libkeymancore1.symbols

```diff
@@ -8,7 +8,7 @@ libkeymancore.so.1 libkeymancore1 #MINVER#
  km_core_context_items_dispose@Base 17.0.195
  km_core_context_length@Base 17.0.195
  km_core_context_set@Base 17.0.195
- km_core_cp_dispose@Base 17.0.263
+ km_core_cu_dispose@Base 17.0.263
```

Comment:
Technically, renaming an API method requires an API version bump (in `CORE_API_VERSION.md` and renaming this file). Additionally this line should be changed to the current version:

```suggestion
 km_core_cu_dispose@Base 18.0.28
```